.travis/mageia/travis-tasks.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+#Exit on failures
+set -e
+
+set -x
+
+
+JOB_NAME=${TRAVIS_JOB_NAME:- Mageia 7}
+
+arr=($JOB_NAME)
+os_name=${arr[0]:-Mageia}
+release=${arr[1]:-7}
+
+COMMON_MESON_ARGS="-Dtest_dirty_git=${DIRTY_REPO_CHECK:-true}"
+
+
+pushd /builddir/
+
+# Build the v1 and v2 code under GCC and run standard tests
+meson --buildtype=debug \
+      -Dbuild_api_v1=true \
+      -Dbuild_api_v2=true \
+      $COMMON_MESON_ARGS \
+      travis
+
+set +e
+ninja -C travis test
+if [ $? != 0 ]; then
+    cat travis_v1/meson-logs/testlog.txt
```

Comment:
This should be `travis`, not `travis_v1`. There was a bug in the centos version originally, but that should have been fixed a few days ago. Make sure you are rebased atop the latest master branch.